src/templates/dev_pipeline.handlebars

```diff
@@ -142,12 +142,12 @@
                       {{#exists pr 'issue_id'}}
                         {{#eq pr.status 'open'}}
                               <a href="https://github.com/duckduckgo/zeroclickinfo-{{repo}}/issues/{{pr.issue_id}}" class="one-line" id="pr" title="{{format_time last_update}}">
-                                {{timeago last_update 0}}
```

Comment:
@jagtalon You're not even supposed to use 0 here. It should just have no arguments. 
Options are the elements to render after the helper, and they never get passed, so it's not a problem to not pass a value for the from_beta value. Look at the overview and issues handlebars.